src/map.cpp

```diff
@@ -1835,6 +1839,11 @@ void ServerMap::save(ModifiedState save_level)
 		infostream<<"Blocks modified by: "<<std::endl;
 		modprofiler.print(infostream);
 	}
+
+	auto end_time = std::chrono::steady_clock::now();
+	std::chrono::duration<double> elapsed_seconds = end_time - start_time;
+	m_save_time_counter->increment(elapsed_seconds.count() * 1000 * 1000 * 1000);
```

Comment:
This should rather be measured using the `porting` methods, which also give you higher precision.